community/cypher/cypher-compiler-3.0/src/test/scala/org/neo4j/cypher/internal/compiler/v3_0/planner/logical/NodeHashJoinPlanningIntegrationTest.scala

```diff
@@ -60,34 +60,6 @@ class NodeHashJoinPlanningIntegrationTest extends CypherFunSuite with LogicalPla
     result should equal(expected)
   }
 
-  test("Should build plans with leaves for both sides if that is requested by using hints") {
```

Comment:
Well... If you look at the test, it's asserting a pretty dumb plan.  The Selection operator on top of the idx-seek-expand (old lines 79 and 80), is doing the exact same thing as the index seek on the RHS (line 82). Not sure why this was being asserted, but IDP correctly does not produce this plan.

I'd say that if it's important to support this pattern (two index seeks for a single pattern relationship), which is entirely possible, we need to teach the planner new tricks.